CONTRIBUTING.md

```diff
@@ -0,0 +1,216 @@
+
+Finally, set up a link between your local clone and the official repository (`upstream`):
+
+```bash
+~$ cd netpyne
+~$ git remote add upstream https://github.com/Neurosim-lab/netpyne.git
+~$ git fetch --all
+~$ git remote -v   #Shows your tracked repositories.
+```
+
+### Basic git commands
+
+Learning to work with git can take a long time, because it is a complex and powerful tool for managing versions of files across multiple users, each of whom have multiple copies of the codebase. We’ve already seen in the setup commands above a few useful git commands:
+
+* `git clone <URL_OF_REMOTE_REPO>` (make a local copy of a repository).
+* `git remote add <NICKNAME_OF_REMOTE> <URL_OF_REMOTE_REPO>` (connect a local copy to an additional remote).
+* `git fetch --all` (get the current state of connected remote repos).
+
+Other commands that you will undoubtedly need related to branches can be found [here](https://help.github.com/en/articles/about-branches). Branches represent multiple copies of the codebase within a local clone or remote repo. Branches are typically used to experiment with new features while still keeping a clean, working copy of the original codebase that you can switch back to at any time. The default branch of any repo is always called `master`, and it is recommended that you reserve the `master` branch to be that clean copy of the working `upstream` codebase. 
+
+In Neurosim-lab/netpyne repo, you must not work in the `master` branch, there is a branch called `development` for contribuiting. Therefore, if you want to add a new change, you should first synchronize your local `development` branch with the `upstream` repository, then create a new branch based on `development` and [check it out](https://git-scm.com/docs/git-checkout) so that any changes you make will exist on that new branch:
+
+```bash
+ ~$ git checkout development            # switch to local development branch
+ ~$ git fetch upstream                  # get the current state of the remote upstream repo
+ ~$ git merge upstream/development      # synchronize local development branch with remote upstream development branch
+ ~$ git checkout -b new-branch          # create local branch "new-branch" and check it out
+```
+
+> **Alternative**
+> You can save some typing by using `~$ git pull upstream/development` to replace the `fetch` and `merge` lines above.
+
+> **Note:** You could synchronize your local `master` branch with the `upstream` repository to have the update version, but you **_must not_** work in any change on the `master` branch.

```

Comment:
You could synchronize your local `master` branch with that of the `upstream` repository to have the updated version, but you **_must never_** make any changes on the `master` branch.

Reply:
:+1:  I change for:
> **Note:** You could synchronize your local `master` branch with that of the `upstream` repository to have the updated version, but you **_must never_** make any changes on the `master` branch.